src/components/WebinarsAndGuides/index.tsx

```diff
@@ -59,9 +69,34 @@ export const WebinarLayout: FunctionComponent<Props> = ({
     }
     useHubSpot(hubSpotConfig)
 
+    const webinarCustomerMobileHeroStyle = { background: `url('${webinarCustomerMobileBg.src}')` }
+    const webinarCustomerHeroStyle = { background: `url('${webinarCustomerBg.src}')` }
+    const webinarProductMobileHeroStyle = { background: `url('${webinarProductMobileBg.src}')` }
+    const webinarProductHeroStyle = { background: `url('${webinarProductBg.src}')` }
+    const guideHeroStyle = { background: `url('${guideBg.src}')` }
+    const guideMobileHeroStyle = { background: `url('${guideMobileBg.src}')` }
+
+    const determineHeroStyle = (): { background: string } => {
+        if (isWebinarPg) {
+            // Customer-based Webinar
+            if (customer) {
+                return isMdOrDown ? webinarCustomerMobileHeroStyle : webinarCustomerHeroStyle
+            }
+            // Product-based Webinar
+            return isMdOrDown ? webinarProductMobileHeroStyle : webinarProductHeroStyle
+        }
+        if (!isWebinarPg) {
+            // Guides
+            return isMdOrDown ? guideMobileHeroStyle : guideHeroStyle
+        }
+        return webinarProductHeroStyle
+    }
+
     return (
         <>
-            <section className={styles.hero}>
+            {/* eslint-disable-next-line react/forbid-dom-props */}
+            <section style={determineHeroStyle()} className={classNames('bg-cover', !isWebinarPg && 'text-white')}>
```

Comment:
Guides/Webinar assets are collated here now, have to do some determining since there are 6 variations of bg-images based on screen size, webinar, or customer-based webinar. Let me know what you think of this solution @bretthayes